fe_plugins/auditloader/src/main/java/org/apache/doris/plugin/audit/AuditLoaderPlugin.java

```diff
@@ -143,8 +143,9 @@ private void assembleAudit(AuditEvent event) {
         auditBuffer.append(event.isQuery ? 1 : 0).append("\t");
         auditBuffer.append(event.feIp).append("\t");
         // trim the query to avoid too long
-        int maxLen = Math.min(2048, event.stmt.length());
-        String stmt = event.stmt.substring(0, maxLen).replace("\t", " ");
+        // use `getBytes().length` to get real byte length
+        int maxLen = Math.min(2000, event.stmt.getBytes().length);
```

Comment:
Could you replace `2000` with a configuration or a `static final` variable?

Reply:
done